.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+- mkdir -p ${HOME}/.sbt
+- docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v ${PWD}:/usace-flood-geoprocessing
```

Comment:
Although it should be noted that `modellab-geoprocessing`, which also didn't use Docker in production, used Docker for CI https://github.com/azavea/modellab-geoprocessing/blob/develop/.travis.yml. But this could very well be a carry over from an earlier project.